RunLog/RunLog/Sources/Presentation/Log/View/DetailLogView.swift

```diff
@@ -0,0 +1,303 @@
+//
+//  DetailLogView.swift
+//  RunLog
+//
+//  Created by 도민준 on 3/17/25.
+//
+
+import UIKit
+import MapKit
+import SnapKit
+import Then
+
+final class DetailLogView: UIView {
+    
+    // MARK: - UI Components 선언
+    
+    /// 전체 스크롤 뷰 (콘텐츠 전체를 스크롤하기 위한 컨테이너)
+    private let scrollView = UIScrollView().then {
+        $0.showsVerticalScrollIndicator = false
+        $0.alwaysBounceVertical = true
+    }
+    
+    /// 스크롤뷰 내부 컨텐츠를 담는 뷰
+    private let contentView = UIView().then { _ in }
```

Comment:
```suggestion
    private let contentView = UIView()
```
혹시 { _ in }을 남겨놓으신 이유가 있으실까요? 2차 작업때 해당 뷰를 작업하시는 건가 싶어 질문드립니다!

Reply:
쓰일지 안쓰일지 몰라서 일단 저렇게 비워놓기만 했습니다.